crates/ghost_actor/Cargo.toml

```diff
@@ -13,7 +13,7 @@ repository = "https://github.com/holochain/lib3h"
 
 [dependencies]
 crossbeam-channel = "=0.3.8"
-holochain_tracing = { version = "=0.0.2", git = "https://github.com/holochain/holochain-tracing" }
+holochain_tracing = { version = "=0.0.3", git = "https://github.com/holochain/holochain-tracing" }
```

Comment:
does it work for publishing to have both of these in here? I guess we can find out next time we release : )

Reply:
both what?

Reply:
if someone is using ghost_actor as a dependency, cargo will look at these dependencies, and either pull it from github or the version from crates.io... this makes it seem ambiguous.

Reply:
Ah. @ddd-mtl - yeah, we need to remove the `git =` part of this. `edge` is [broken currently](https://github.com/holochain/lib3h/commits/edge) since a new release got pushed to holochain-tracing - because it's going to github expecting to receive version `0.0.2` but it finds `0.0.3` there.